backend/src/packages/chats/chat.repository.ts

```diff
@@ -70,8 +74,8 @@ class ChatRepository implements Repository {
     });
   }
 
-  public update(): Promise<unknown> {
-    return Promise.resolve(null);
+  public async update({ id, updatedAt }: UpdateChatPayload): Promise<void> {
+    await this.chatModel.query().patchAndFetchById(id, { updatedAt });
   }
```

Comment:
```suggestion
  publiс update({ id, updatedAt }: UpdateChatPayload): Promise<ChatEntity> {
    return this.chatModel.query().patchAndFetchById(id, { updatedAt }).execute();
  }
```

Reply:
@v0ldemar01 But this method doesn't return ChatEntity. Actually, it isn't needed, I guess, as I want only to update updatedAt column in chat. This is it.

Reply:
Yep, ChatModel. But if you are using pathAndFetch then the function will return result